modules/performance/extension/siddhi-execution-performance/component/src/main/java/org/wso2/extension/siddhi/execution/throughput/CalculatePerformanceStreamProcessorExtension.java

```diff
@@ -288,7 +334,7 @@ protected void process(ComplexEventChunk<StreamEvent> streamEventChunk, Processo
 
             default:
                 log.error("executionType should be either throughput or latency or both "
-                                  + "but found" + " " + executionType);
+                        + "but found" + " " + executionType);
```

Comment:
+ "but found" + " " can be changed to + "but found " +

Reply:
Fixed https://github.com/wso2/product-sp/pull/880/commits/bcb8b40e0439f8be16498c95eb514241f9e723df